articles/install-guide/csinstall.md

```diff
@@ -135,7 +136,86 @@ Of course, you can also build and run Q# programs from the command line by simpl
 
         You should see the following output: `Hello quantum world!`
 
+## Q# with other .NET languages <a name="dotnet"></a>
+
+Q# is compiled to C# for the purposes of local simulation. This makes them very easy to combine. However, you can also use other .NET languages like F# or VB\.NET, although the process is less straightforward.
+
+### Develop with Q# + F# creating a Q# library
+
+You can't mix Q# and F# files in the same project like we do with Q# and C#. To combine F# with Q# you need to create a Q# library which will yield a .csproj project with only Q# files in it. Then you need to reference it from a purely F# application.
+
+1. Pre-requisites
+
+    - [.NET Core SDK 3.1 or later](https://www.microsoft.com/net/download)
+
+1. Create a Q# library `QuantumCode` and write your quantum code in it.
+
+1. Create an F# application (in this case a console app targeting .NET Core) `FsharpDriver`.
+
+1. Add a reference to the Q# library to the F# application.
+
+   You can use [Reference Manager](https://docs.microsoft.com/visualstudio/ide/how-to-add-or-remove-references-by-using-the-reference-manager) in Visual Studio to do that, or you can add the reference from the command line:
+
+    ```PowerShell
+    PS>  dotnet add .\FsharpDriver\FsharpDriver.vbproj reference .\QuantumCode\QuantumCode.csproj
+    ```
+   
+   This will transitively include the [`Microsoft.Quantum.Development.Kit` NuGet package](https://www.nuget.org/packages/Microsoft.Quantum.Development.Kit) to the F# application.
+   You will not be writing any Q# code in `FsharpDriver`, but you will need to use functionality provided by the Quantum Development Kit to create a quantum simulator to run your quantum code on, and to define data types used to pass the parameters to your quantum program.
+1. Write the classical driver in `FsharpDriver`:
```

Comment:
I would add a sentence here explaining what the driver is doing here, i.e., it's a running a quantum operation that takes an array of integers as an argument and returns an integer (or a boolean? it's not clear from the code)